Client/Frontend/Settings/TranslationSettingsController.swift

```diff
@@ -98,17 +80,23 @@ class TranslationSettingsController: ThemedTableViewController {
             if indexPath.row == 0 {
                 cell.textLabel?.text = Strings.SettingTranslateSnackBarSwitchTitle
                 cell.detailTextLabel?.text = Strings.SettingTranslateSnackBarSwitchSubtitle
-                cell.detailTextLabel?.numberOfLines = 2
+                cell.detailTextLabel?.numberOfLines = 4
```

Comment:
Ensure plenty of lines for translation (beyond 4 full lines the font will shrink).